### ISSUE TITLE
Error installing django-CMS with Django 1.7.

### ISSUE THREAD

**Issue by rjhelms** [divio/django-cms#4080]
I am attempting to install django CMS on a Windows 7 environment, but am running into an issue when using Django 1.7 or stable.

Installation seems to fail when installing django-select2-py3 - by modifying \djangocms_installer\install\__init__.py to not run pip in quiet mode, I get this error:

```
Collecting django-select2-py3 (from djangocms-link==1.5)
  Using cached Django-Select2-Py3-4.2.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "C:\Users\Rob\AppData\Local\Temp\pip-build-bd0gs3ca\django-select2-py
3\setup.py", line 166, in <module>
        long_description=read("README.md").decode('utf-8'),
    AttributeError: 'str' object has no attribute 'decode'

    ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in C:\Users\Rob\
AppData\Local\Temp\pip-build-bd0gs3ca\django-select2-py3
```

This occurs when leaving all installation options at the defaults, but does not occur when I specify Django 1.6 in the install options.

I am running Windows 7, Python 3.4.3, and using virtualenvwrapper to create virtual environments.

**Comment by yakky** [comment]
django-select2-py3 is currently broken. django-select2 just released a python 3 compatible version, we will update dependencies soon

**Comment by yakky** [comment]
This should have been fixed in djangocms-link 1.6
